content/guides/models/track/runs/multiple-runs-per-process.md

```diff
@@ -0,0 +1,168 @@
+   * Ideal when you want to break sequential sub-processes into separate individual runs.
+   * Default behaviour for non notebook environments.
+- `return_previous`: Return the most recent, unfinished run.
+   * This option does not create a new run.
+   * Default behaviour for notebook environments.
+
+{{% alert  %}}
+W&B does not support `create_new` mode for integrations that assume a single global run, such as Hugging Face Trainer, Keras callbacks, and PyTorch Lightning. If you use these integrations, you should run each sub-experiment in a separate process.
+{{% /alert %}}
+
+## Specifying `reinit`
+
+<!-- There are several ways to create and manage multiple runs in a single Python process: -->
+
+- Use `wandb.init()` with the `reinit` argument directly:
+   ```python
+   import wandb
+   wandb.init(reinit="<create_new|finish_previous|return_previous>")
+   ```
+- Use `wandb.init()` and pass a `wandb.Settings` object to the `settings` parameter. Specify `reinit` in the `Settings` object:
+
+   ```python
+   import wandb
+   wandb.init(settings=wandb.Settings(reinit="<create_new|finish_previous|return_previous>"))
+   ```
+
+- Use `wandb.setup()` to set the `reinit` option globally for all runs in the current process. This is useful if you want to configure the behavior once and have it apply to all subsequent `wandb.init()` calls in that process.
+
+   ```python
+   import wandb
+   wandb.setup(wandb.Settings(reinit="<create_new|finish_previous|return_previous>"))
+   ```
+
+- Specify the desired value for `reinit` in an environment variable `WANDB_REINIT`. Defining an environment variable applies the reinit option to `wandb.init()` calls.
+
+   ```bash
+   export WANDB_REINIT="<create_new|finish_previous|return_previous>"
+   ```
+
+The following code snippet shows a high level overview how to set up W&B to create a new run each time you call `wandb.init()`:
+
+```python
+import wandb
+
+wandb.setup(wandb.Settings(reinit="create_new"))
+
+with wandb.init() as experiment_results_run:
+   for ...:
+      with wandb.init() as run:
+         # The do_experiment() function logs fine-grained metrics
+         # to the given run and then returns result metrics that
+         # you want to track separately.
+         experiment_results = do_experiment(run)
+
+         # After each experiment, log its results to a parent
+         # run. Each point in the parent run's charts corresponds
+         # to one experiment's results.
+         experiment_results_run.log(experiment_results)
+```
+## Example: Concurrent processes
+
+Suppose you want to create a primary process that remains open for the script's entire lifespan, while periodically spawning short-lived secondary processes without finishing the primary process. This may occur if you want to keep a primary run (like training a model) active while computing evaluations or other tasks in separate runs.
```

Comment:
```suggestion
Suppose you want to create a primary process that remains open for the script's entire lifespan, while periodically spawning short-lived secondary processes without finishing the primary process. This may occur if you want to keep a primary run such as a training run active, while computing evaluations or other tasks in separate runs.
```